.github/workflows/main.yml

```diff
@@ -442,7 +442,7 @@ jobs:
       - lint-vcxagency-node
       - lint-vcxagency-client
       - lint-easy-indysdk
-    runs-on: ubuntu-16.04
+    runs-on: ubuntu-latest
```

Comment:
Though quite low risk here, in general `latest` is not a good practice. The latest right now is `ubuntu-20.04`, let's state it explicitly.

Reply:
@Patrik-Stas Actually, latest = 18.04 and in this case I don't see it as a bad practice (new version won't break anything because it's just a docker host and you always get updated OS - less worries about old unpatched OS with potential vulnerabilities)

<img width="592" alt="Screen Shot 2020-08-12 at 10 37 51" src="https://user-images.githubusercontent.com/36594792/89994490-7d0eb400-dc88-11ea-8f52-8b126d6cb8f4.png">

<img width="592" alt="Screen Shot 2020-08-12 at 10 37 51" src="https://user-images.githubusercontent.com/36594792/89994329-46d13480-dc88-11ea-9f1d-24c96cb06c60.png">